analyzers/nodejs/nodejs.go

```diff
@@ -261,6 +263,18 @@ func (a *Analyzer) Analyze() (graph.Deps, error) {
 	return yarn.FromProject(filepath.Join(a.Module.BuildTarget, "package.json"), filepath.Join(a.Module.BuildTarget, "yarn.lock"))
 }
 
+// sanitizeBuildTarget ensures that legacy behavior stays intact but users are warned if it is implemented.
+func (a *Analyzer) sanitizeBuildTarget() {
+	if strings.HasSuffix(a.Module.BuildTarget, "/package.json") {
+		log.Warn("Specifiying the package.json file as a module's target in fossa's config file is no longer supported. Only the directory is neccessary.")
+		a.Module.BuildTarget = strings.TrimSuffix(a.Module.BuildTarget, "/package.json")
+	}
+	if strings.HasSuffix(a.Module.BuildTarget, "/") {
+		log.Warn("The provided fossa config file is malformed. Ensure there are no trailings /'s in any of the specified modules' targets.")
```

Comment:
Telling them it's malformed before specifying _how_ it's malformed is a bit redundant. I'd prefer: `Trailing slashes in module targets are not supported.`